Source/Request.swift

```diff
@@ -326,15 +326,17 @@ extension Request: CustomDebugStringConvertible {
                 let cookieStorage = session.configuration.httpCookieStorage,
                 let cookies = cookieStorage.cookies(for: URL), !cookies.isEmpty
             {
-                let string = cookies.reduce("") { $0 + "\($1.name)=\($1.value ?? String());" }
+                let string = cookies.reduce("") { $0 + "\($1.name)=\($1.value);" }
                 components.append("-b \"\(string.substring(to: string.characters.index(before: string.endIndex)))\"")
             }
         }
 
-        var headers: [NSObject: AnyObject] = [:]
+        var headers: [String: String] = [:]
 
         if let additionalHeaders = session.configuration.httpAdditionalHeaders {
-            for (field, value) in additionalHeaders where field != "Cookie" {
+            for (field, value) in additionalHeaders {
+                guard let field = field as? String, let value = value as? String, field != "Cookie" else { continue }
+
                 headers[field] = value
             }
         }
```

Comment:
Works, and is cleaner. Hopefully we'll be able to change back when they fix `httpAdditionalHeaders`.